locales-pending/dashboard-premium.ftl

```diff
@@ -189,3 +189,14 @@ exposure-card-description-info-for-sale-action-needed-dashboard = This site is p
 exposure-card-description-info-for-sale-in-progress = We’ve started our auto-removal process of <data_broker_profile>this profile</data_broker_profile> to protect your information. <removal_info>Removals typically take 7-14 days</removal_info>.
 exposure-card-description-info-for-sale-fixed = As a { -brand-monitor-plus } member, we’ve <data_broker_profile>removed this profile</data_broker_profile> for you and will continually monitor to make sure they don’t add you back.
 exposure-card-description-info-for-sale-fixed-manually-fixed =  You marked this profile as fixed. Be sure you’ve followed all instructions on <data_broker_profile>the site</data_broker_profile> to ensure they remove your personal info.
+
+## CSAT survey banner for subscribers
+
+survey-csat-question = How satisfied are you with your { -product-name } experience?
```

Comment:
We have too many brand terms…

```
-product-name = Mozilla Monitor
-brand-fx-monitor = Mozilla Monitor
-brand-mozilla-monitor = Mozilla Monitor
```

In case, the choice here would be among `{ -brand-fx-monitor }` (9 strings), `{ -product-name }` (15 strings), `{ -brand-mozilla-monitor }` (3 strings). 

`-brand-mozilla-monitor` is probably the latest, definitely not `-brand-monitor`(that's just `Monitor`, not `Mozilla Monitor`).

Reply:
Ah right, sorry, I was looking at line 201 (which has `-product-short-name`) first, and then attached this to the topmost line.

Reply:
So to confirm: Should we rather use `-brand-mozilla-monitor` because it is the latest instance and the others are going to be deprecated?

Reply:
I'd go with `-brand-mozilla-monitor`, at least it clear what replaces it without searching for the definition.

Reply:
Sounds good, thanks! Updated in [82ed4c9](https://github.com/mozilla/blurts-server/pull/4421/commits/82ed4c9295a7cd24ca262fd4759e52573a1494e6).